### ISSUE TITLE
DOCUMENTATION: SFAL (Standardized Framework Abstraction Library)

### ISSUE THREAD

**Issue by hassanhabib** [hassanhabib/The-Standard#251]
- We need to identify the highest priority items in an existing framework to abstract away:

In terms of higher priority items we are thinking of the following:
- Most commonly used types (`String`, `IQueryable`, `List`, `Exception`, `Exception` ... )
- Most commonly used routines (`ToString()`, `Streams` ...)
- The Language itself `C#`?

**Comment by hassanhabib** [comment]
Actionable Items:
- @cjdutoit will take a look at how these SDKs are created and how we can modify them. (@hassanhabib will try this too)
- @tehwardy will take a look at the most common types and routines. (@hassanhabib will gather some as well)

**Comment by LBoullosa** [comment]
just a thought,

could be seen SFAL like multiples SPAL to the .NET framework?

**Comment by hassanhabib** [comment]
@LBoullosa yes. it really is just a collection of SPALs for the .NET library 100%
